src/http/router.ts

```diff
@@ -1213,23 +1211,19 @@ export const getRouter = (client: CLClient, options: ICLClientOptions): express.
                 const result = await clRunner.recognize(turnContext)
 
                 if (result) {
-                    // Did I select the expected action?
-                    if (turnValidation.actionHashes[0] && !Utils.actionHasHash(result.scoredAction.actionId, turnValidation.actionHashes[0], appDefinition.actions)) {
-                        validity = CLM.TranscriptValidationResultType.CHANGED
-                    }
 
                     // Get conversation reference
                     const conversationReference = BB.TurnContext.getConversationReference(activity)
                     if (!conversationReference) {
-                        validity = CLM.TranscriptValidationResultType.TEST_FAILED
-                        break
+                        res.send(null)
```

Comment:
Seems weird to send `null`. Does this send nothing or actual null value?
I don't even see example of this on docs: http://expressjs.com/en/api.html#res.send

At least hope it doesn't mean error or something.

Reply:
Hmm...  Seems to work.  Could send back an empty obj instead